.pre-commit-config.yaml

```diff
@@ -9,7 +9,6 @@ repos:
       - id: trailing-whitespace
       - id: debug-statements
       - id: check-merge-conflict
-      - id: check-docstring-first
```

Comment:
This is removed because the sphinx-style of documenting variables (docstring beneath declaration) is incompatible with this hook.